app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -198,7 +199,7 @@ function algoliaPaginate(tag){
   } else if (homeEl.dataset.feed === "latest") {
     paginationArticlesIndex = client.initIndex('ordered_articles_by_published_at_<%= Rails.env %>');
   } else {
-    searchObj.filters = ('published_at_int > ' + homeEl.dataset.articlesSince);
+    searchObj.tagFilters = ('published_at_int > ' + homeEl.dataset.articlesSince);
```

Comment:
I'm not following how `searchObj.tagFilters` went from an array of strings to a string. Is it's type array of string or string?

Reply:
There's two params here, I renamed filters to tagFilters (since it was more apt) which is an array, and introduced a variable called filters which is a string.